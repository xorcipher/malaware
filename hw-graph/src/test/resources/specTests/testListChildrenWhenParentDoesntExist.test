# Tests that the getChildren method works properly for a graph in which the graph does not contain the specified node.

# Return all the children of a given node within a given graph in a List.

CreateGraph graph

AddNode graph A
AddNode graph B

AddEdge graph A B edgeA

ListChildren graph C