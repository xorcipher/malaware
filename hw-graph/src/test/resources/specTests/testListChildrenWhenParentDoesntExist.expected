# Tests that the getChildren method works properly for a graph in which the graph does not contain the specified node.

# Return all the children of a given node within a given graph in a List.

created graph graph

added node A to graph
added node B to graph

added edge edgeA from A to B in graph

Exception: java.lang.IllegalArgumentException: Graph does not contain node: C